src/main/java/org/ambraproject/wombat/service/ArticleArchiveServiceImpl.java

```diff
@@ -71,17 +70,20 @@
    * {@inheritDoc}
    */
   @Override
-  public Map<?, ?> getArticleDoisPerMonth(Site site, String year, String month) throws IOException {
-    SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
-
+  public Map<?, ?> getArticleDoisPerMonth(Site site, String year, String month) throws IOException, ParseException {
     Calendar startDate = Calendar.getInstance();
-    startDate.set(Integer.parseInt(year), Arrays.asList(MONTHS).indexOf(month), 1);
```

Comment:
This is the ultimate source of the bug. `Arrays.asList(MONTHS).indexOf(month)` was an incorrect use of this method, which expects EG Calendar.DECEMBER

Making a ticket to refactor this usage of `MONTHS` and `getMonthsForYear`. It's awkward and I think we can tighten it up.